pkg/micro-ecc/psa_uecc/p192.c

```diff
@@ -32,7 +32,8 @@ psa_status_t psa_generate_ecc_p192r1_key_pair(  const psa_key_attributes_t *attr
 
     const struct uECC_Curve_t *curve = uECC_secp192r1();
 
-    ret = uECC_make_key(pub_key_buffer, priv_key_buffer, curve);
+    pub_key_buffer[0] = 0x04;
+    ret = uECC_make_key(pub_key_buffer+1, priv_key_buffer, curve);
```

Comment:
Would you mind adding a comment to these lines to explain this magic number? Maybe referring to https://github.com/kmackay/micro-ecc/blob/master/README.md#point-representation and https://arm-software.github.io/psa-api/crypto/1.1/api/keys/management.html#key-formats

(or maybe even better use a single define at the top of the file with `0x04` and add the comment there)

Reply:
I added a comment to explain the magic number. This is how it is was done in the atca driver.